python/mlc_chat/compiler/parameter/huggingface_loader.py

```diff
@@ -90,7 +105,21 @@ def load(self) -> Iterator[Tuple[str, NDArray]]:
         mlc_names = _loading_order(self.extern_param_map, self.torch_to_path)
         for mlc_name in tqdm(mlc_names):
             param = self._load_mlc_param(mlc_name)
-            yield mlc_name, param
+            if self.quantize_param_map:
```

Comment:
handle the case that `param` is not in `self.quantize_param_map` - meaning no quantization